packages/help-text/README.md

```diff
@@ -149,12 +149,12 @@ When the content associated to the element is disabled, use the `disabled` attri
 
 Good, descriptive help text includes 1-2 short sentences of information such as:
 
--   An overall description of an input field or controls
--   Hints for what kind of information needs to be inputted or selected
--   Specific formatting examples or requirements
+- An overall description of an input field or controls
```

Comment:
This must have been a lint change, but I think it's right, so I left it. Happy to revert if folks think this is incorrect.